gubernator/github/classifier.py

```diff
@@ -206,12 +201,24 @@ def classify(events, statuses=None):
                 'xrefs': list of builds referenced (by GCS path),
             }
     '''
+    if events_iterator is not None:
+        events = next(events_iterator)
```

Comment:
Assuming you need to explicitly handle the events_iterator is None case separately from not events_iterator, doesn't this need to handle next raising StopIteration?

Also is the for loop below going to duplicate things?

Reply:
Seems unaddressed

Reply:
Github reviews are bad. I split classify into a streaming and non-streaming version.